src/components/raw-capacity-card/raw-capacity-card.tsx

```diff
@@ -0,0 +1,154 @@
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import * as React from 'react';
+import * as _ from 'lodash';
+import classNames from 'classnames';
+import { useTranslation } from 'react-i18next';
+import { ChartDonut, ChartLabel } from '@patternfly/react-charts';
+import { useDashboardPrometheusQuery as usePrometheusQuery } from "@console/dynamic-plugin-sdk/provisional";
+import {
+  DashboardCard,
+  DashboardCardBody,
+  DashboardCardHeader,
+  DashboardCardTitle,
+} from '@console/dynamic-plugin-sdk/provisional';
+import { humanizeBinaryBytes } from '../../humanize';
+import { CAPACITY_BREAKDOWN_QUERIES_ODF,StorageDashboardQuery } from '../../constants/queries';
+import {
+  useK8sWatchResource,
+} from "@console/dynamic-plugin-sdk/api";
+import {GetFlashSystemResource} from '../../constants/resources';
+import { StorageInstanceKind } from '../../types';
+import './raw-capacity-card.scss';
+
+const colorScale = ['#0166cc', '#d6d6d6'];
+
+const RawCapacityCard: React.FC<any> = (props)  => {
+  const { t } = useTranslation();
+  const [data, loaded, loadError] = useK8sWatchResource<StorageInstanceKind>(GetFlashSystemResource(props?.match?.params?.name, props?.match?.params?.namespace));
+  const name= loaded && !loadError? data?.[0]?.metadata.name: undefined;
+  
+  const [totalCapacity, ,] = usePrometheusQuery(
+    CAPACITY_BREAKDOWN_QUERIES_ODF(name, StorageDashboardQuery.TotalCapacity),
+    humanizeBinaryBytes
+  );
+  const [usedCapacity, ,] = usePrometheusQuery(
+    CAPACITY_BREAKDOWN_QUERIES_ODF(name, StorageDashboardQuery.TotalUsedCapacity),
+    humanizeBinaryBytes
+  );
+  const [availableCapacity, ,] = usePrometheusQuery(
+    CAPACITY_BREAKDOWN_QUERIES_ODF(name, StorageDashboardQuery.TotalFreeCapacity),
+    humanizeBinaryBytes
+  );
+
+  const donutData = [
+    { x: 'Used', y: usedCapacity.value, string: usedCapacity.string },
+    {
+      x: 'Available',
+      y: availableCapacity.value,
+      string: availableCapacity.string,
+    },
+  ];
+
+  return (
+    <DashboardCard>
+      <DashboardCardHeader>
+        <DashboardCardTitle>{t('Physical Capacity Overview')}</DashboardCardTitle>
+      </DashboardCardHeader>
+      <DashboardCardBody className="flashsystem-raw-usage__container">
+        {loaded && !loadError && (
+          <>
+            <div className="flashsystem-raw-usage__item flashsystem-raw-usage__legend">
+              <ChartLegend
+                fill={colorScale[0]}
```

Comment:
I feel we need to change color when used capacity is crossing a certain threshold for warning and danger. We are using #f0ab00 for warning and #c9190b for danger. @afreen23 thoughts?